### PR TITLE
cmd/scollector: win.service.* metrics

### PR DESCRIPTION
**Pull request by gbrayut**
Also cleanup default process and process_dotnet settings.
This is currently running as a second instance on ny-web01 pointed at devbosun

### CODE REVIEW

File: cmd/scollector/collectors/processes_windows.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func c_windows_processes() (opentsdb.MultiDataPoint, error) {
 	}
 
 	var svc_dst []Win32_Service
-	var svc_q = wmi.CreateQuery(&svc_dst, `WHERE Started=true`)
+	var svc_q = wmi.CreateQuery(&svc_dst, "")
```
Comment (alienth) [inline cmd/scollector/collectors/processes_windows.go]: I'm not sure what is going on here. Would you care to drop the sick knowledge beats on me?

Comment (kylebrandt) [inline cmd/scollector/collectors/processes_windows.go]: @alienth It reflects the Object (svc_dst) and creates a WMI sql like query. You can specify a where filter as a string.

Comment (kylebrandt) [inline cmd/scollector/collectors/processes_windows.go]: See https://msdn.microsoft.com/en-us/library/aa394606%28v=vs.85%29.aspx for more about WQL.

Comment (kylebrandt) [inline cmd/scollector/collectors/processes_windows.go]: @alienth Oh also I use https://wmie.codeplex.com/ to explore these. Then we have some linqpad C# which generates the Go structs from C#. Somewhere in the repo

Comment (gbrayut) [inline cmd/scollector/collectors/processes_windows.go]: https://github.com/bosun-monitor/bosun/blob/master/util/Windows_WMI_Metadata.linq is a linqpad script that can do some simple code generation for us. Try running it from ny-rdp01.

Basically we use the github.com/stackexchange/wmi library to use reflection to create a WQL query based on a struct.

Comment (kylebrandt) [inline cmd/scollector/collectors/processes_windows.go]: https://github.com/StackExchange/wmi/blob/master/wmi.go#L393

**Pull request merged**